orc8r/protos/service_registry.proto

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package magma.orc8r;
+
+import "orc8r/protos/common.proto";
+
+option go_package = "magma/orc8r/lib/go/protos";
+
+
+// --------------------------------------------------------------------------
+// ServiceRegistry supports dynamic service discovery.
+// --------------------------------------------------------------------------
+service ServiceRegistry {
+
+  // ListAllServices returns the service name of all services in the registry.
+  rpc ListAllServices (Void) returns (ListAllServicesResponse) {}
```

Comment:
In general I still prefer e.g. an empty `ListAllServicesRequest` defn over `Void`, though for this and other orc8r-internal grpc servicers it's not super relevant